modmesh/app/sample_mesh.py

```diff
@@ -89,6 +89,18 @@ def help_3dmix(set_command=False):
     if set_command:
         view.mgr.pycon.command = cmd.strip()
 
+def help_sciwork(set_command=False):
```

Comment:
Quite nice.  But can we use "solvcon", or "modmesh"?  This project is not part of sciwork, while it happens to participate in sciwork activities.

Reply:
Sure! I can swift to use "solvcon" or "modmesh". I think I'll first try to use 2D to represent it and then think about how to optimize for using 3D.